l10n_br_resource/models/resource_calendar.py

```diff
@@ -0,0 +1,253 @@
+                                   date_start <= data_referencia <= data_end
+
+        :return retorna True ou False
+        """
+        eh_feriado = self.data_eh_feriado(data_referencia)
+        dia_antes = data_referencia - timedelta(days=1)
+        dia_depois = data_referencia + timedelta(days=1)
+
+        dia_antes_eh_segunda = \
+            True if dia_antes.weekday() == 0 or self.data_eh_feriado(
+                dia_antes
+            ) else False
+        dia_depois_eh_sexta = \
+            True if dia_depois.weekday() == 4 or self.data_eh_feriado(
+                dia_depois
+            ) else False
+
+        return eh_feriado and (dia_antes_eh_segunda or dia_depois_eh_sexta)
+
+    @api.multi
+    def data_eh_dia_util(self, data=datetime.now()):
+        """Verificar se data é dia util.
+        :param datetime data: Se nenhuma data referencia for passada
+                              verifique o dia de hoje.
+        :return boolean True: Se for dia útil
+                        False: Se Não for dia útil
+        """
+        return not self.data_eh_feriado(data) and data.weekday() <= 4 or False
+
+    @api.multi
+    def quantidade_dias_uteis(
+            self, data_inicio=datetime.now(), data_fim=datetime.now()):
+        """Calcular a quantidade de dias úteis em determinado período.
+        :param datetime data_inicio: Se nenhuma data referencia for passada
+                                   verifique o dia de hoje.
+        :param datetime data_fim: Se nenhuma data referencia for passada
+                                   verifique o dia de hoje.
+        :return int: Quantidade de dias úteis
+        """
+        dias_uteis = 0
+        while data_inicio <= data_fim:
+            if self.data_eh_dia_util(data_inicio):
+                dias_uteis += 1
+            data_inicio += timedelta(days=1)
+
+        return dias_uteis
+
+    @api.multi
+    def proximo_dia_util(self, data_referencia=datetime.now()):
+        """Retornar o próximo dia util.
+        :param datetime data_referencia: Se nenhuma data referencia for passada
+                                   verifique se amanha é dia útil.
+        :return datetime Proximo dia util apartir da data referencia
+        """
+        data_referencia += timedelta(days=1)
+        while data_referencia:
+            if self.data_eh_dia_util(data_referencia):
+                return data_referencia
+            data_referencia += timedelta(days=1)
+
+    @api.multi
+    def get_dias_base(self, data_from=datetime.now(), data_to=datetime.now()):
+        """Calcular a quantidade de dias que devem ser remunerados em
+        determinado intervalo de tempo.
+        :param datetime data_from: Data inicial do intervalo de tempo.
+               datetime data_end: Data final do intervalo
+        :return int : quantidade de dias que devem ser remunerada
+        """
+        return 30 - data_from.day + 1
```

Comment:
Essa fórmula não funciona para admissão.
Se o funcionário for admitido no dia 20/05 , deverá ser remunerado 12 dias.

Reply:
https://github.com/abgfbr/odoo-brazil-hr/blob/8.0/l10n_br_resource/models/resource_calendar.py#L216